etc/kayobe/compute.yml

```diff
@@ -179,6 +179,15 @@
 # compute_qemu_conf_extra.
 #compute_qemu_conf:
 
+# Whether to enable libvirt SASL authentication. Default is true.
+#compute_libvirt_enable_sasl:
+
+# libvirt SASL password. Default is unset.
+# NOTE: You can use:
+#   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/generate-stackhpc-secrets.yml
+# to generate secrets_libvirt_sasl_password.
+compute_libvirt_sasl_password: "{{ secrets_libvirt_sasl_password }}"
```

Comment:
Probably needs to be in the ci-aio environment?

Reply:
Can we keep it in the base config? It isn't used unless you set kolla_enable_nova_libvirt_container to false.